functions/src/morningEmailCentral/morningEmailCentral.spec.js

```diff
@@ -0,0 +1,44 @@
+import * as firebaseTesting from '@firebase/testing';
+import morningCentralOriginal from './index';
+
+const context = { timestamp: Date.now() };
+
+const morningCentral = functionsTest.wrap(morningCentralOriginal);
+const adminApp = firebaseTesting.initializeAdminApp({
+  projectId,
+  databaseName: projectId,
+});
+
+describe('morningEmailCentral Schedule Cloud Function (schedule:onRun)', () => {
+  beforeEach(async () => {
+    // Clean database before each test
+    await firebaseTesting.clearFirestoreData({ projectId });
+  });
+
+  it('adds user to users_public on create event', async () => {
+    const userUid = '123ABC';
+    await adminApp
+      .firestore()
+      .doc('system_settings/notifications')
```

Comment:
Do tests intentionally not use the constants?

Reply:
Yeah - I have had it be nice to have tests have everything they need in them other than what is being pulled in to test. That said, we can use constants here if we find that to be easier